package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "NODE_ENV=production node server/server.js",
-    "dev:server": "ts-node-dev --respawn --pretty --project server/tsconfig.json server/server.ts",
+    "dev:server": "tsc client/scripts/auth.ts && ts-node-dev --respawn --pretty --project server/tsconfig.json server/server.ts",
```

Comment:
I'm not sure this change is required, `npm run dev:client` should compile `auth.ts` too.

Reply:
I run the only the server, I thought the server is serving the client. 
I realise now that I'm supposed to run both the dev: commands in a separate terminal